### DISCUSSION TITLE
GitHub Actions to deploy GitHub Pages failed.

### DISCUSSION THREAD

**Discussion by kimlimjustin** [facebook/docusaurus#5211] [Q&A] [answered]
When I tried to docusaurus manually, it works fine, however, when I try to deploy using GitHub Actions, it failed.

At first, I try to modify and use the code from the [documentation](https://docusaurus.io/docs/deployment#triggering-deployment-with-github-actions) and this happens: 
[![Screenshot from 2021-07-23 13-46-48](https://user-images.githubusercontent.com/61906684/126746508-ab98b9c6-f856-4b1f-a786-c274b5304337.png)](https://github.com/kimlimjustin/xplorer/runs/3135604941?check_suite_focus=true)

Later on, I tried to modify the code to use `GITHUB_PASS` and `GITHUB_USER` env instead of the `webfactory/ssh-agent`, and it gives me the following error:

![Screenshot from 2021-07-23 13-49-47](https://user-images.githubusercontent.com/61906684/126746746-0fef2570-5f4a-4e51-b590-0488b05036b4.png)

My code is on [GitHub](https://github.com/kimlimjustin/xplorer/tree/ci) too!

Can anyone please help me? Thanks

**Comment by slorber** [comment, marked as answer]
Can't help you much, not too familiar with all those GH deployment problems.
In general I'd recommend using Netlify or Vercel, much better platforms imho.
We also have a #deployment chan in Discord now

**Reply by kimlimjustin** [reply]
Hmm, sure! Thanks for the reply tho! I'll try vercel!
